docs/learn/learn-nft-projects.md

```diff
@@ -0,0 +1,190 @@
+---
+id: learn-nft-projects
+title: NFT projects on Polkadot and Kusama
+sidebar_label: NFT Projects
+description: An explanation of the NFT projects in the ecosystem.
+keywords: [NFT, non-fungible token, NFT 2.0]
+slug: ../learn-nft-projects
+---
+
```

Comment:
Could you add the list of the projects here, so that anyone who visits the page is able to see the full list right away and then click on the project they are interested in.

```suggestion
## List of NFT Projects

- [Astar](#Astar)

```